sigar_darwin.go

```diff
@@ -420,7 +420,12 @@ func kern_procargs(pid int,
 			return fmt.Errorf("Error reading args: %v", err)
 		}
 		pair := bytes.SplitN(chop(line), delim, 2)
-		env(string(pair[0]), string(pair[1]))
+
+		if len(pair) == 2 {
```

Comment:
It seems more common in Go to check for the error case and omit the `else` (i.e. `if len != 2 { return error }`).

Reply:
+1 will make the changes